tests/integration.rs

```diff
@@ -910,7 +911,16 @@ fn check_sequential_events(expected_events: &[&MetaEvent], event_file: &str) ->
         }
     }
 
-    idx == len
+    let ret = idx == len;
+
+    if !ret {
+        eprintln!(
+            "\n\n==== Start 'check_sequential_events' failed ====\n\nexpected_events={:?}\nactual_events={:?}\n\n==== End 'check_sequential_events' failed ====",
```

Comment:
Perhaps this could be a multiline string?

Reply:
Will fix it.

Reply:
Thanks, there are are some others in the same file..

Reply:
Good catch. All fixed.